src/js/components/Box/index.d.ts

```diff
@@ -38,7 +38,7 @@ export interface BoxProps {
   flex?: 'grow' | 'shrink' | boolean | { grow?: number; shrink?: number };
   fill?: FillType;
   focusIndicator?: boolean;
-  gap?: GapType;
+  gap?: GapType | { row?: string; column?: string };
```

Comment:
```suggestion
  gap?: GapType | { row?: GapType; column?: GapType };
```
realizing we could be a bit more specific here.

Reply:
good call!